src/components/CreateNonFungiblePage/index.tsx

```diff
@@ -145,25 +144,12 @@ export default function CreateNonFungiblePage() {
             >
               {state.step === 'collection_select' ? 'Create' : 'Next'}
             </MinterButton>
-            <StatusModal
-              isOpen={isOpen}
-              onClose={() => {
-                onClose();
-                setLocation('/collections');
-                dispatch(setStatus({ method: 'mintToken', status: 'ready' }));
-                dispatch(clearForm());
-              }}
-              onRetry={() => {
-                dispatch(clearError({ method: 'mintToken' }));
-                dispatch(mintTokenAction());
-              }}
-              onCancel={() => {
-                onClose();
-                dispatch(clearError({ method: 'mintToken' }));
-                dispatch(setStatus({ method: 'mintToken', status: 'ready' }));
-              }}
-              status={status}
-            />
+            <div style={{position: 'absolute', bottom: 0, right: 0, width: 'auto', display: 'flex', flexFlow: 'column nowrap', justifyContent:'flex-end', alignItems: 'flex-end', borderTopRightRadius: '5px', boxShadow: '-2px -2px 20px #3333', padding: '5px 0 0 5px'}}>
```

Comment:
Bug: This doesn't navigate back to the collection after the creation of the token anymore.

Reply:
I'm not sure I follow, it seems to work for me @ricklove